Source/SwiftLintFramework/Rules/RuleConfigurations/RegexConfiguration.swift

```diff
@@ -43,6 +44,10 @@ public struct RegexConfiguration: RuleConfiguration, Equatable {
 
         regex = try NSRegularExpression.cached(pattern: regexString)
 
+        if let includedString = configurationDict["included"] as? String {
+          included = try NSRegularExpression.cached(pattern: includedString)
```

Comment:
we use 4-space indentation everywhere else in this project. Could you do that here too?